pallets/runtime/tests/src/storage.rs

```diff
@@ -721,11 +721,21 @@ pub fn default_portfolio_btreeset(did: IdentityId) -> BTreeSet<PortfolioId> {
     iter::once(PortfolioId::default_portfolio(did)).collect::<BTreeSet<_>>()
 }
 
+/// Returns a vector that contains default portfolio for the identity.
+pub fn default_portfolio_vec(did: IdentityId) -> Vec<PortfolioId> {
+    vec![PortfolioId::default_portfolio(did)]
+}
+
 /// Returns a btreeset that contains a portfolio for the identity.
 pub fn user_portfolio_btreeset(did: IdentityId, num: PortfolioNumber) -> BTreeSet<PortfolioId> {
     iter::once(PortfolioId::user_portfolio(did, num)).collect::<BTreeSet<_>>()
 }
```

Comment:
I think we can remove this as vector version is in use right now.

Reply:
I'll just let it be there for now. It's only part of the tests. This change is a temporary change to add compatibility with the polkadot UI. I'll prefer to revert back to btreesets when we build our own UIs.

Reply:
It will be a long way 😜